t/unit/backends/test_database.py

```diff
@@ -45,7 +45,7 @@ def _create_request(self, task_id, name, args, kwargs,
         )
         if task_protocol == 1:
             body, headers, _, _ = hybrid_to_proto2(msg, msg.body)
-            properties = {}
+            properties = None
```

Comment:
what about None or {}

Reply:
I changed this field to reflect the problem and create a more edge-case like scenario when the properties field are present but with `None` value.